.pipelines/ci.yml

```diff
@@ -851,10 +851,9 @@ stages:
         architecture: 'x64'
       displayName: "Use Python 3.12"
 
-    # iOS xcframework build doesn't work with CMake 3.25.1, pin to 3.25.0
     - script: |
-        python -m pip install cmake==3.25.0
-      displayName: "Install CMake 3.25.0"
```

Comment:
I think it was caused by openCV stuff

Reply:
sorry, I don't remember. if it works with a later version then that's good.

Reply:
validated it. I think openCV was the reason.